video/cloud-client/analyze/analyze.py

```diff
@@ -69,14 +67,16 @@ def analyze_explicit_content(path):
 
 def analyze_faces(path):
     """ Detects faces given a GCS path. """
-    video_client = videointelligence_v1beta2.VideoIntelligenceServiceClient()
-    features = [enums.Feature.FACE_DETECTION]
+    video_client = videointelligence.VideoIntelligenceServiceClient()
+    features = [videointelligence.enums.Feature.FACE_DETECTION]
 
-    config = types.FaceDetectionConfig(include_bounding_boxes=True)
-    context = types.VideoContext(face_detection_config=config)
+    config = videointelligence.types.FaceDetectionConfig(
+        include_bounding_boxes=True)
+    context = videointelligence.types.VideoContext(
+        face_detection_config=config)
 
     operation = video_client.annotate_video(
-        path, features, video_context=context)
+        path, features=features, video_context=context)
     print('\nProcessing video for face annotations:')
 
     while not operation.done():
```

Comment:
Since you're here, can you drop this while loop and just use `operation.result()`?

Reply:
Done.